Web of Science Nextgen.js

```diff
@@ -2,14 +2,14 @@
 	"translatorID": "4a3820a3-a7bd-44a1-8711-acf7b57d2c37",
 	"label": "Web of Science Nextgen",
 	"creator": "Abe Jellinek",
-	"target": "^https://www\\.webofscience\\.com/",
+	"target": "^https://(www\\.webofscience\\.com/|webofscience\\.clarivate\\.cn)/",
```

Comment:
Looks like there's an extra `/` in the non-China one that might break this for everyone outside of China.

Reply:
I was careless, you were right